[CI] Reduce distributed test timeout to 60s

kwen2501 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #141168

Pulling a PR to test viability. 
Today's timeout is 300s, which could waste quite some machine time if a hang happens in CI.

cc @H-Huang @awgu @wanchaol @fegin @fduwjj @wz337 @wconstab @d4l3k @c-p-i-o

Differential Revision: [D66275756](https://our.internmc.facebook.com/intern/diff/D66275756)

kwen2501 committed:
[CI] Reduce distributed test timeout to 60s

[ghstack-poisoned]

kwen2501 commented:
cc @clee2000 @huydhn 
I wonder if there is a way to check test time distribution? That would also help us set the timeout threshold -- not too long, not too flaky.

clee2000 commented:
> cc @clee2000 @huydhn I wonder if there is a way to check test time distribution? That would also help us set the timeout threshold -- not too long, while not too flaky.

We have a database of the past 7 days of test data, you can request access by following https://fb.workplace.com/groups/4571909969591489/posts/8684358118346633/?comment_id=8684466965002415&reply_comment_id=8688241291291649

kwen2501 commented:
@kwen2501 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D66275756).

kwen2501 committed:
Update on "[CI] Reduce distributed test timeout to 60s"


Pulling a PR to test viability. 
Today's timeout is 300s, which could waste quite some machine time if a hang happens in CI.

cc H-Huang awgu wanchaol fegin fduwjj wz337 wconstab d4l3k c-p-i-o

Differential Revision: [D66275756](https://our.internmc.facebook.com/intern/diff/D66275756)

[ghstack-poisoned]

kwen2501 committed:
Update on "[CI] Reduce distributed test timeout to 60s"


Pulling a PR to test viability. 
Today's timeout is 300s, which could waste quite some machine time if a hang happens in CI.

cc H-Huang awgu wanchaol fegin fduwjj wz337 wconstab d4l3k c-p-i-o

Differential Revision: [D66275756](https://our.internmc.facebook.com/intern/diff/D66275756)

[ghstack-poisoned]

kwen2501 committed:
Update on "[CI] Reduce distributed test timeout to 60s"


Pulling a PR to test viability. 
Today's timeout is 300s, which could waste quite some machine time if a hang happens in CI.

cc H-Huang awgu wanchaol fegin fduwjj wz337 wconstab d4l3k c-p-i-o

Differential Revision: [D66275756](https://our.internmc.facebook.com/intern/diff/D66275756)

[ghstack-poisoned]

kwen2501 committed:
Update on "[CI] Reduce distributed test timeout to 60s"


Pulling a PR to test viability. 
Today's timeout is 300s, which could waste quite some machine time if a hang happens in CI.

cc H-Huang awgu wanchaol fegin fduwjj wz337 wconstab d4l3k c-p-i-o

Differential Revision: [D66275756](https://our.internmc.facebook.com/intern/diff/D66275756)

[ghstack-poisoned]

kwen2501 commented:
@pytorchbot merge

huydhn commented:
@pytorchbot revert -m 'Sorry for reverting your change but I think we missed inductor tests' -c nosignal

distributed/_composable/fsdp/test_fully_shard_compile.py::TestFullyShardCompile::test_transformer_backend_inductor_fullgraph_True [GH job link](https://github.com/pytorch/pytorch/actions/runs/11964436110/job/33358048772) [HUD commit link](https://hud.pytorch.org/pytorch/pytorch/commit/e8de8f3969bf935442378efd125442de90e78431)

Some are timing out after 60s
